client/src/views/header/HeaderPage.vue

```diff
@@ -94,6 +94,29 @@
                 :icon="search"
               />
             </ion-button>
+            <div
+              v-if="!isMobile() && securityWarningsCount > 0 && isSmallDisplay"
+              id="trigger-checklist-button"
+              class="topbar-right-button__item unread"
+              @click="openSecurityWarningsModal()"
+              ref="checklistSecurityButton"
+            >
+              <ion-icon :icon="checkmarkCircle" />
+              <div class="checklist-security-levels">
+                <span
+                  class="security-level"
+                  :class="{ 'security-level--done': securityWarningsCount < 3 }"
+                />
+                <span
+                  class="security-level"
+                  :class="{ 'security-level--done': securityWarningsCount <= 1 }"
```

Comment:
You can use `< 2` or `=== 1` since the only possible cases here can be 1, 2 or 3